sdk/communication/azure-communication-identity/tests/_shared/helper.py

```diff
@@ -10,6 +10,7 @@ class URIIdentityReplacer(RecordingProcessor):
     def process_request(self, request):
         import re
         request.uri = re.sub('/identities/([^/?]+)', '/identities/sanitized', request.uri)
+        request.uri = re.sub('^(.*?)\\.communication.azure.com', 'https://sanitized.communication.azure.com', request.uri)
```

Comment:
Is there a way to just replace the regex instead of the whole string? The thing is that we may want to know if were tested against an INT resource or not even when the url was sanitized. As an example an INT URL may be something like: https://resourcename.int.communication.azure.com

Reply:
Yes, we need to consider the scenario for testing in INT environment.

Reply:
Good catch, I updated this to only sanitize the resource.